src/FunctionApps/SilverTransforms/Transform/__init__.py

```diff
@@ -0,0 +1,66 @@
+import logging
+import os
+
+import pymongo
+
+import azure.functions as func
+
+from Transform.fhir import get_patient_records, write_patient_records
+from Transform.geo import get_smartystreets_client
+from Transform.transforms import transform_record
+
+
+CACHE_EXPIRATION = 60 * 60 * 24 * 7  # seconds * minutes * hours * days = 1 week
```

Comment:
Curious, what's the rationale for setting the cache expiration so high? Is it just under the assumption there will be roughly one week between receiving batches of data?

Reply:
Just a random guess here until we know more about what the data looks like (and how we're going to use it) - I know SmartyStreets gets expensive fast, so we may want to bring it up even higher.

Reply:
Totally fair.